### PR TITLE
select largest submodel produced by colmap for splats generation

### PR DESCRIPTION
**Pull request by isumygin-sc**
colmap could generate several sub models for a single video, we selected model 0 previously. 
Now we check the size of images.bin files in each sub model and select biggest

### CODE REVIEW

**Review by willeastcott — APPROVED** [review]

File: convert.py [hunk 1]
```diff
@@ -73,21 +73,46 @@
...
-    ### Convert model to text format so we can read cameras
-    convert_cmd = (colmap_command + " model_converter \
-        --input_path " + args.source_path + "/distorted/sparse/0 \
-        --output_path "  + args.source_path + "/distorted/sparse/0 \
-        --output_type TXT")
-    exit_code = os.system(convert_cmd)
-    if exit_code != 0:
-        logging.error(f"Convert failed with code {exit_code}. Exiting.")
-        exit(exit_code)
+
```
Comment (slimbuck) [inline convert.py:79]: This block should probably remain inside `if not args.skip_matching:` block

File: convert.py [hunk 2]
```diff
@@ -73,21 +73,46 @@
...
+        index = i
+
+    i += 1
+
+str_index = str(index)
+distorted_sparse_path = args.source_path + "/distorted/sparse/" + str_index
+
+
+### Convert model to text format so we can read cameras
+convert_cmd = (colmap_command + " model_converter \
```
Comment (slimbuck) [inline convert.py]: Currently gs-generation/app.js extracts cameras from `/distorted/sparse/0/images.txt`. With this change that won't work anymore. I'd suggest specifying output location of `/distorted/sparse/` here and load then load cameras from `/distorted/sparse/images.txt` instead.

File: convert.py (hunk 1 above)
Comment (isumygin-sc) [reply convert.py:79]: i moved conversion to the later stage and introduce a new parameter for the script

File: convert.py (hunk 2 above)
Comment (isumygin-sc) [reply convert.py]: it's updated in monorepo

**Review by slimbuck — APPROVED** [review]

**Pull request merged**